index.bs

```diff
@@ -2737,13 +2870,26 @@ The [=remote end steps=] with |command parameters| are:
   1. If the implementation is unable to create a new browsing context for any
      reason then return [=error=] with [=error code=] [=unsupported operation=].
 
+  1. Let |user context| be the [=default user context=].
+
+  1. If |command parameters|[<code>userContext</code>] is present and is not null:
+
+    1. Let |user context id| be |command parameters|[<code>userContext</code>].
+
+    1. If the [=user context=] with the [=user context id=] equal to |user context id| does
+      not exist in the [=remote end=], return [=error=] with [=error code=] [=invalid argument=].
+
+    1. Set |user context| to the [=user context=] with the [=user context id=] equal
+      to |user context id|.
+
   <!-- This is based on step 5 of https://w3c.github.io/webdriver/#new-window,
        but without using the "current browsing context" concept. -->
   1. Create a new [=top-level browsing context=] by running the [=window open
-     steps=] with <var ignore>url</var> set to "<code>about:blank</code>",
-     <var ignore>target</var> set to the empty string, and
-     <var ignore>features</var> set to "<code>noopener</code>". Which OS window the new [=/browsing context=]
-     is created in depends on |type| and |reference context|:
+     steps=] in the |user context| with <var ignore>url</var> set to
+     "<code>about:blank</code>", <var ignore>target</var> set to the empty
+     string, and <var ignore>features</var> set to "<code>noopener</code>".
```

Comment:
For `type=window`, I think we agree that the `user_context` provided in `browsingContext.create` will only impact the initial tab of this window. If additional tabs are created in the same window with `browsingContext.create` + `type=tab`, they will not be assigned by default to the user context used to create the window.

Reply:
Good point, I think we can and should validate that explicitly. I have added a validation step.

Reply:
Or let me think about it, maybe it's simpler to change the default.

Reply:
I have ended up doing both improvements. PTAL